list_etexts.php

```diff
@@ -18,21 +18,26 @@
     $title = _("Completed Gold E-Texts");
     $state = SQL_CONDITION_GOLD;
     $info = _("Below is the list of Gold e-texts that have been produced on this site. Gold e-texts are books that have passed through all phases of proofreading, formatting, and post-processing. They have been submitted to Project Gutenberg and are now available for your enjoyment and download.");
+    $rss_content = "posted";
 } elseif ($x == "s") {
     $type = "Silver";
     $title = _("In Progress Silver E-Texts");
     $state = SQL_CONDITION_SILVER;
     $info = _("Below is the list of Silver e-texts that have almost completed processing on our site. Silver e-texts are books that have passed through all phases of proofreading and formatting and are now in the post-processing phase. Post-processing is the final assembly stage in which one volunteer performs a series of checks for consistency and correctness before the e-book is submitted to Project Gutenberg for your enjoyment and download.");
+    $rss_content = "postprocessing";
 } elseif ($x == "b") {
     $type = "Bronze";
     $title = _("Now Proofreading Bronze E-Texts");
     $state = SQL_CONDITION_BRONZE;
     $info = _("Below is the list of Bronze e-texts that are currently available for proofreading on this site. Bronze e-texts are what our newest volunteers see and what you can work on now by logging in. These e-texts are in the initial stages of proofreading where everyone has a chance to correct any OCR errors which may be found. After going through a number of other phases, the e-text then goes to an experienced volunteer for final assembly (post-processing), after which the e-text is submitted to Project Gutenberg for your enjoyment and download.");
+    $rss_content = "proofing";
 } else {
     die("x parameter must be 'g', 's', or 'b'. ('$x')");
 }
 
-output_header($title);
+output_header($title, SHOW_STATSBAR, array(
+    "head_data" => "<link rel='alternate' type='application/rss+xml' href='$code_url/feeds/backend.php?content=$rss_content&type=rss' title='$title' />",
```

Comment:
$title needs to be escaped with `attr_safe()` in case the translated version has non-attribute-safe characters. Also, consider pulling the array into its own variable and passing that rather than breaking the open_header() call onto multiple lines (totally valid, but most of the code uses the assign-to-variable-and-pass-it approach for additional header attributes).

Reply:
Good catch! Fixed both in 1d8ed3d3601b81fadf3c699c6ef650e870e27495 :)